tests/snuba/api/endpoints/test_group_details.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import, print_function
+
+import mock
+
+from sentry.models import Environment
+from sentry.testutils import APITestCase, SnubaTestCase
+
+
+class GroupDetailsTest(APITestCase, SnubaTestCase):
+    def test_multiple_environments(self):
+        group = self.create_group()
+        self.login_as(user=self.user)
+
+        environment = Environment.get_or_create(group.project, 'production')
+        environment2 = Environment.get_or_create(group.project, 'staging')
+
+        url = u'/api/0/issues/{}/?enable_snuba=1'.format(group.id)
+
+        from sentry.api.endpoints.group_details import tsdb
```

Comment:
Can we remove this?

Reply:
i think it's needed on line 23?

Reply:
Don't know how I missed that, ignore me!